### PR TITLE
amend to allow the prompt to be skipped

### PR DESCRIPTION
**Pull request by moloko**
Resolves https://github.com/adaptlearning/adapt_framework/issues/1509.

### CODE REVIEW

**Review by tomgreenfield — APPROVED** [review]

**Review by brian-learningpool — APPROVED** [review]

File: js/adapt-contrib-bookmarking.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 define([
-    'coreJS/adapt'
+    'core/js/adapt'
 ], function(Adapt) {
 
     var Bookmarking = _.extend({
```
Comment (oliverfoster) [inline js/adapt-contrib-bookmarking.js]: Can you change to:
```javascript
var Bookmarking = Backbone.Controller.extend({

});

return new Bookmarking();

```

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: js/adapt-contrib-bookmarking.js (hunk 1 above)
Comment (tomgreenfield) [reply js/adapt-contrib-bookmarking.js]: Would need the framework dependency bumping to `2.0.18` if this goes in.

Comment (oliverfoster) [reply js/adapt-contrib-bookmarking.js]: If you say so 👍  https://github.com/adaptlearning/adapt_framework/releases/tag/v2.0.18

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-bookmarking",
-  "version": "2.0.4",
+  "version": "2.0.5",
```
Comment (moloko) [inline bower.json]: oops this should actually be 2.1.0 now cause this is a new feature

**Review by moloko — COMMENTED** [review]
need to set version to 2.1.0

**Review by tomgreenfield — APPROVED** [review]

**Review by chetan-hajare — APPROVED** [review]

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**